templates/ts-apollo-postgres-backend/doc.graphql

```diff
@@ -0,0 +1,147 @@
+fragment NoteFields on Note {
```

Comment:
Is this a trailing files and should not be committed?

Reply:
Sorry, this was me doing manual testing